tcmu-runner.spec

```diff
@@ -1,3 +1,29 @@
+%global _hardened_build 1
+
+# uncomment and add '%' to use the prereltag for pre-releases
+# %%global prereltag qa3
+
+# without librbd1 dependency
+# if you wish to compile an rpm without librbd1...
```

Comment:
Probably you mean to say.
\# if you wish to exclude rbd handler at rpm

Same at other places too

Reply:
Done